indico/modules/events/export.yaml

```diff
@@ -391,6 +397,8 @@ export:
   # conference menu
   MenuEntry:
     skipif: ROW.type.name == 'plugin_link'
+    fks:
+      - MenuEntryPrincipal.menu_entry_id
```

Comment:
This triggers the export if MenuEntryPrincipal objects if they are referenced in a MenuEntry that has been exported